res/layout/video_preview.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Renaming it as preview_video_fragment could be confusing because this layout is currently being used in the preview video fragment and also in the full screen mode, when any fragment is shown.